src/content/docs/new-relic-only/tech-writer-style-guide/article-templates/html-template.mdx

```diff
@@ -73,16 +82,26 @@ For all other lists, use unordered (bulleted) lists `ul` or [clamshells.](/docs/
       </td>
 
       <td>
-        More detailed information about it
+        More detailed information about another thing
       </td>
     </tr>
   </tbody>
 </table>
+```
 
-## For more help [#more_help]
+Tables created in Markdown use three or more hyphens `---` to create each column’s header, and pipes `|` to separate each column. For example:
+
+```
+| Item       | Requirements
+------------ | ------------
+| Some thing | More detailed information about it
+| Another thing | More detailed information about another thing
+```
+
+## Last section (For more help) [#last_section]
 
-This is sample text for the **For more help** section. If there are no links to include, remove the text but leave the **For more help** H2. The Docs site automatically fills customized footer content.
+Always add one final section to your doc called `For more help`. A bulleted list of helpful resources is automatically generated in this section when your doc is published.
 
-Additional documentation resources include:
+By default, the list of resources does not appear when your doc is a draft. You do not need to add anything here manually.
 
-* `<a href>Link to relevant page</a>` (useful description)
+## For more help [#more_help]
```

Comment:
Minor nit:

Our templates will traverse the MDX document and look for a heading of `For more help` when trying to decide if the default for more help should be included or not. If it finds the header, it doesn't include it (that way it can be customized in a document).

I'd recommend removing this, or at least mentioning this behavior.

Reply:
Oh! The previous section that I labeled "Last section (for more help) intended to be that explanatory part. then, the real "For more help" that immediately follows I intended to be the "For more help" specifically for this doc so that the bullets would automatically fill. I had asked Bradley C. to peer edit this. Maybe he can suggest a way to make this more clear. Thanks for your input, Jerel!

Reply:
Oh gotcha! I didn't read that close enough 😅 . Thanks for clarifying!